### ISSUE TITLE
Consider a slightly different license

### ISSUE THREAD

**Issue by choldgraf** [data-8/textbook#36]
Hey all - was just looking at the license that the textbook is published under.

https://creativecommons.org/licenses/by-nc-nd/4.0/

It looks fine to me, with one exception: there's a "no derivatives" clause in there.

I'm a bit worried that this will discourage others from adapting the textbook for their own purposes. As we are starting to encourage other universities/colleges/etc to adapt Data 8 material for their own uses, I wonder if this will be an impediment to them making the necessary changes for their specific course requirements.

Curious if the team has had a discussion about this. I'll defer to others on the final decision but wanted to bring it up in case it hadn't been considered.

**Comment by stefanv** [comment]
I just ran into this when I browsed [Matthew Brett's fork of this material](https://github.com/matthew-brett/dsfe), and noticed that he had to branch from 64b20f0 because of the license change.

Can someone comment on whether this was a purposeful decision?

**Comment by davidwagner** [comment]
This is up to John & Ani, who wrote almost all of the material in the textbook.

**Comment by choldgraf** [comment]
@stefanv or @matthew-brett out of curiosity, do you have an idea for a license that would be less-restrictive that could still work. I believe that the original authors wanted to make sure that others wouldn't modify the content and still call it "the data 8 textbook" etc, and that the original content would be tied back to the Data 8 course. I agree the current version is too restrictive, but what would be a good alternative?

**Comment by choldgraf** [comment]
maybe @fperez has thoughts on this as well. Fernando what license are you using for the DS100 textbook?

**Comment by ryanlovett** [comment]
@choldgraf The DS100 textbook's [README](https://github.com/DS-100/textbook) says it is licensed under [CC BY-NC-ND 4.0](https://creativecommons.org/licenses/by-nc-nd/4.0/).

**Comment by choldgraf** [comment]
ah ok - I think that's pretty near the same - my guess is that the problem in question is the "no-derivatives" clause, since many people would want to remix this material for use in their own courses (or if they have shorter-then-a-semester courses)

**Comment by fperez** [comment]
Ultimately the decision isn't up to me - I'm using the textbook in DS100 but I haven't had any hand in its authorship.  I'm just a guy on github here :)

I do think that -ND clauses in content make it more difficult to really reuse the content elsewhere, for reasons including those articulated by Matthew above.  The impact of these materials could be even larger if others can more flexibly adapt them.

FWIW, I think it might be viable to come up with a model similar to what Mozilla uses for Firefox: the code is open source but the name is trademarked and has different restrictions - this leads Debian to ship IceWeasel instead - Firefox renamed to comply with the Mozilla conditions.  RedHat uses the same thing, which is what makes CentOS basically RedHat minus the names.

*If* the issue is regarding the protection of the name/brand for Data8/100, then I could imagine licensing the content under say CC-BY-NC or CC-BY but adding a restriction on the name itself (though I don't know if that would require formal trademarking first).

But the authors (Ani and John) may have had a different reason for selecting the name, so I won't speak for them.  Happy to discuss it further, here or on campus, obviously.

**Comment by papajohn** [comment]
Ani and I prefer to have direct contact with authors who wish to revise and
republish the Data 8 textbook. We have often granted permissions
individually for remixing, but have no plans to let anyone remix and
republish the book at this time.

**Comment by matthew-brett** [comment]
Ah - that is a shame.   Here in the UK, we're thinking about how to expand our data science teaching quickly, and that closes a door to something that would have been a very good foundation.

**Comment by choldgraf** [comment]
ah that is too bad - I think that this ND clause drastically limits the impact that the textbook could have (and the "Data 8 @ other institutions" model more generally). However it is you + Ani's call.

**Comment by papajohn** [comment]
I'm not sure I get why this license is such a problem. If folks want to use
a subset of the textbook, they can easily link to the parts that are
relevant to their students. If they want to redirect the "interact" links,
we support that already. If they want to write supplemental material, of
course they're welcome to publish that material anywhere they want.

If there are other use cases that involve remixing, feel free to ask and
we'll probably say yes on an individual basis.

**Comment by matthew-brett** [comment]
@papajohn - I wrote some of the reasons and use-cases in the comment up at https://github.com/data-8/textbook/issues/36#issuecomment-435600939 .  I'm teaching a course, that I was planning to base on the Berkeley course, and, in practice I've found that it's very difficult to stick so completely to the Data 8 track that it makes sense to point people to individual chapters.    I can see, from multiple readings, how carefully you have built up the exposition - with the consequence that taking a chapter here or there simply doesn't work.  Honestly I think that would be true for any well-written textbook.

I think you will find that it doesn't really help in reducing the impact of the ND, saying yes on an individual basis, because the remix remains ND, and my impression is that many of us in this space find it uncomfortable writing something that can't easily be re-used.
